src/partcad/runtime_python_conda.py

```
@@ -444,19 +552,26 @@ def once_conda_locked_attempt(self):
                         shell=False,
                         encoding="utf-8",
                     )
-                    _, stderr = p.communicate()
+                    stdout, stderr = p.communicate()
 
                 if not stderr is None and stderr.strip() != "":
```

Ruff (0.16.2)

[error] Test for object identity should be `is not`

Convert to `is not`

(E714)